pombola/south_africa/templates/core/person_detail.html

```diff
@@ -7,297 +7,236 @@
+          <ul>
+            {% for fax in fax_contacts %}
+            <li>{{ fax }}</li>
           {% endfor %}
           </ul>
         {% endif %}
-      {% endwith %}
 
-        <h3>Formerly</h3>
-          <ul class="former-positions positions">
-          {% if past_positions %}
-            {% for position in past_positions %}
-              <li>{% include "core/person_detail_position.html" %}
-                {% if position.start_date %}
-                  from {{ position.start_date }}
-                {% endif %}
-                until {{ position.end_date }}
-              </li>
+        </div>
+        <div class="column">
+          {% if address_contacts %}
+          <h3>Address</h3>
+          <ul>
+            {% for address in address_contacts %}
+              <li>{{ address }}</li>
             {% endfor %}
-          {% else %}
-            <li>No former positions recorded.</li>
-        {% endif %}
-        </ul>
+          </ul>
+          {% endif %}
 
+          {% if object.constituency_offices %}
+          <h3>Constituency Offices</h3>
+          <ul>
+            {% for office in object.constituency_offices %}
+              <li><a href="{{ office.get_absolute_url }}">{{ office.name }}</a></li>
+            {% empty %}
+              <li>Not associated with any constituency offices</li>
+            {% endfor %}
+          </ul>
+          {% endif %}
+          <h3>Party</h3>
+          <ul>
+            {% for party in object.parties_and_coalitions %}
+              <li><a href="{{ party.get_absolute_url }}">{{ party.name }}</a></li>
+            {% empty %}
+              {% if former_parties %}
+                {% for party in former_parties %}
+                  <li><a href="{{ party.get_absolute_url }}">{{ party.name }}</a></li>
+                {% empty %}
+                  <li>Not a member of any party</li>
+                {% endfor %}
+              {% else %}
+                <li>Not a member of any party</li>
+              {% endif %}
+            {% endfor %}
+          </ul>
+
+          {% regroup object.constituencies|dictsort:"kind" by kind as constituency_groups %}
+          {% for entry in constituency_groups %}
+          <h3>{{ entry.grouper }}</h3>
+          <ul>
+            {% for area in entry.list %}
+              <li><a href="{{ area.get_absolute_url }}">{{ area.name }}</a></li>
+            {% endfor %}
+          </ul>
+          {% endfor %}
+
+        </div>
+      </div>
+      <div class="person-summary">
+        <h3>About {{ object.name }}</h3>
+          {% if person.date_of_death %}
+            <p>Died {{ person.date_of_death }}</p>
+          {% endif %}
+          {{ person.summary }}
+      </div>
     </div>
   </div>
-  {% endif %}{% endwith %}
+</div>
+<div class="page-wrapper wrapper">
+  <div class="large-container">
+    <div class="tabs">
+      <ul class="tab-links">
+        {% with past_positions=person.position_set.all.political.currently_inactive %}
```

Comment:
It's a bit confusing that this `with` block opening is in here - I'd put it after `large-container`'s reopening just above, and the `endwith` symmetrically just before that `div` is closed.